MSAL/src/MSALErrorConverter.m

```diff
@@ -92,7 +114,11 @@ + (void)initialize
                              MSIDCorrelationIdKey : MSALCorrelationIDKey,
                              MSIDErrorDescriptionKey : MSALErrorDescriptionKey,
                              MSIDOAuthErrorKey: MSALOAuthErrorKey,
-                             MSIDOAuthSubErrorKey: MSALOAuthSubErrorKey
+                             MSIDOAuthSubErrorKey: MSALOAuthSubErrorKey,
+                             MSIDDeclinedScopesKey: MSALDeclinedScopesKey,
+                             MSIDGrantedScopesKey: MSALGrantedScopesKey,
+                             MSIDUserDisplayableIdkey: @(1),
+                             MSIDBrokerVersionKey: @(1)
```

Comment:
Why these two maps to the same value?

Reply:
Because I forgot to fix them :) Will fix